### PR TITLE
Feature/jacobayu/fonts button

### LINKED ISSUE #2: [Components] Buttons, Typography, and Icons
- Basic button component with color variants
- Global CSS files for fonts and icons

Screen Shot 2023-01-24 at 3 47 33 PM

Screen Shot 2023-01-24 at 3 47 49 PM

Screen Shot 2023-01-24 at 3 48 12 PM

### PR DESCRIPTION
**Pull request by jacobayu**
### Tracking Info

Resolves #2 

Make sure your branch name conforms to: `<feature|staging|bugfix|...>/<username>/<3-4 word description separated by dashes>`. Otherwise, please rename your branch and create a new PR.

### Changes

Added global fonts, icons, and added button component

- TODO

### Testing

Visually looked at fonts and buttons

- TODO

### Confirmation of Change

Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

TODO...

### CODE REVIEW

**Review by Cubified — APPROVED** [review]
LGTM, nice work!

(Added the bold variant of the title font and the color palette as CSS variables)

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -1,2 +1,157 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
+<script>
+  import Button from '../lib/button.svelte';
+</script>
+
+<div>
```
Comment (EmilyMarschallNiswonger) [inline src/routes/+page.svelte]: Maybe make these buttons centered with a row class?

**Review by EmilyMarschallNiswonger — APPROVED** [review]
Looks good!

File: src/routes/+page.svelte (hunk 1 above)
Comment (Cubified) [reply src/routes/+page.svelte]: Resolved!  Added a global CSS class to help with this

**Pull request merged**